content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,329 @@
 
-Once a metric is submitted, there are 3 metric storage types within the Datadog web application:
+- COUNT
+- RATE
+- GAUGE
+- DISTRIBUTION
 
-* `COUNT`
-* `RATE`
-* `GAUGE`
+**Note**: If you submit a metric to Datadog without a type, the metric type appears as `Other` within Datadog. The `Other` metric type cannot be further changed to another in-app type until an initial metric type is submitted.
 
-Refer to the [Metric Type Definition](#metric-type-definition) section below to learn more about the different metrics types available. Also, refer to the [Submission Types and Datadog In-App Types Section](#submission-types-and-datadog-in-app-types) to see how each metric type behaves between its submission and its storage within Datadog.
+## Submission vs. in-app type
 
-## Metric type definition
+Metrics are submitted to Datadog in three main ways:
 
-To better understand the different metrics types, what they represent, and how to modify them within Datadog, consider the following example:
+- [Custom Agent check][4]
+- [DogStatsD][5]
+- [Datadog’s HTTP API][6]
 
-You have two web servers: `server:web_1` and `server:web_2`. Both web servers continuously receive:
+The majority of data that Datadog receives is submitted by the Agent, either through an Agent check or DogStatsD. For these submission methods, a metric’s type determines how multiple values collected on an Agent in [a flush time interval][7] are aggregated. The Agent combines these values into a single representative metric value for that interval. This combined value is stored with a single timestamp in Datadog.
 
-* 10 HTTP requests for the first 30 seconds, then
-* 20 HTTP requests for the next 30 seconds, then
-* 0 HTTP requests for the next 30 seconds.
+Data submitted directly to the Datadog API is not aggregated by Datadog, with the exception of distribution metrics. The raw values sent to Datadog are stored as-is.
 
-### Metric submission types
+Refer to the [Submission and In-App Types](#submission-types-and-datadog-in-app-types) section to see how different metric submission types are mapped to their corresponding in-app types.
+
+## Metric types
+
+### Definition
 
 {{< tabs >}}
 {{% tab "COUNT" %}}
 
-**The `COUNT` metric submission type represents the number of events that occur in a defined time interval, otherwise known as a flush interval**. This number of events can accumulate or decrease over time -- it is not monotonically increasing. A `COUNT` can be used to track the number of requests hitting your webservers or number of errors.
+The COUNT metric submission type represents the total number of event occurrences in one time interval. A COUNT can be used to track the total number of connections made to a database or the total number of requests to an endpoint. This number of events can accumulate or decrease over time—it is not monotonically increasing.
 
```

Comment:
**Note**: A COUNT is different from the RATE metric type, which represents the number of event occurrences normalized per second given the defined time interval.